django_socio_grpc/protobuf/tests/test_json_format.py

```diff
@@ -0,0 +1,18 @@
+from django_socio_grpc.protobuf.json_format import message_to_dict
+from django_socio_grpc.protobuf.tests.protos import test_proto_pb2
+
+
+def test_message_to_dict():
```

Comment:
add comment to the goal of this test. I don't understand about oneof_field2 that do not appear in this test and not sure if in purpose or not